modules/ensemble/lib/widget/input/slider/slider.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:ensemble/widget/helpers/input_wrapper.dart';
+import 'package:flutter/material.dart';
+import 'package:ensemble/framework/action.dart';
+import 'package:ensemble_ts_interpreter/invokables/invokable.dart';
+import 'package:ensemble/util/utils.dart';
+import 'composites/track_style.dart';
+import 'composites/tick_mark_style.dart';
+import 'composites/thumb_style.dart';
+import 'composites/overlay_style.dart';
+import 'composites/value_indicator_style.dart';
+import 'slider_controller.dart';
+import 'slider_state.dart';
+
+class EnsembleSlider extends StatefulWidget
+    with Invokable, HasController<SliderController, SliderState> {
+  static const type = 'Slider';
+
+  EnsembleSlider({Key? key}) : super(key: key);
+
+  final SliderController _controller = SliderController();
+  @override
+  SliderController get controller => _controller;
+
+  @override
+  State<StatefulWidget> createState() => SliderState();
+
+  @override
+  Map<String, Function> getters() {
+    return {
+      'value': () => _controller.value,
+      'min': () => _controller.minValue,
+      'max': () => _controller.maxValue,
+      'trackStyle': () => _controller.trackStyle,
```

Comment:
these are made as composite classes so that binding works in js, but without providing these composite styles in getter, they were not accessible and were giving error `trackStyle property is not found`.

Reply:
ah okay, will check.